app/controllers/api/v2/moves_actions.rb

```diff
@@ -113,9 +114,12 @@ def common_move_attributes
 
     def validate_move_status
       status = move_params.fetch(:attributes, {})[:status]
+      Rails.logger.debug("Validating current status of move - [#{status}]")
       if status.present?
         validator = Moves::StatusValidator.new(status: status, cancellation_reason: move_params.fetch(:attributes, {})[:cancellation_reason], rejection_reason: move_params.fetch(:attributes, {})[:rejection_reason])
-        raise ActiveModel::ValidationError, validator unless validator.valid?
+        valid = validator.valid?
+        Rails.logger.debug("Move status - [#{valid}]")
```

Comment:
Might be clearer to say "Move validation status" so we don't confuse it with `move.status`?

Reply:
It is validating the move.status. Reworded to clarify

Reply:
ahh ok makes sense. Thanks